src/style.css

```diff
@@ -1 +1,81 @@
 /* Styles go here */
+html {
+  background-color: hsl(0, 0%, 90%);
+}
+
+* {
+  padding: 0;
+  margin: 0;
+  border: 0;
```

Comment:
```suggestion
```
redundant to reset padding, margin, border for all selectors. better to do it where it is necessary